jdks.gradle

```diff
@@ -1,25 +1,19 @@
-String compat(String src) {
-   if (src.contains('.')) {
-      src.substring(src.lastIndexOf('.')+1)
-   } else {
-      src
-   }
-}
+def compat = { it.replaceFirst(/^.*\./, '') }
```

Comment:
The regex should be equivalent to the previous code `lastIndexOf('.')` because `.*` is greedy. More specifically `1.2.3.4` will be stripped to `4`.